package.json

```diff
@@ -47,16 +46,13 @@
   },
   "devDependencies": {
     "@tsconfig/node22": "^22.0.1",
-    "@types/mocha": "^10.0.10",
-    "@types/node": "~22.10.7",
-    "chai": "^5.2.0",
+    "@types/node": "^22.12.0",
```

Comment:
We had initially pinned `"~22.10.7"` as the version range so that we wouldn't accidentally introduce new features from minor releases of Node.js that v22.12 didn't have

Reply:
I don't think `@types/node` tracks node versions like that? I could be wrong tho